ruskeyverbs/views.py

```diff
@@ -1,8 +1,58 @@
 from django.shortcuts import render
 from django.http import HttpResponse
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views import generic
+from .models import Verb, Example, PerformancePerExample
 
 # Create your views here.
 
 
 def index(request):
-    return HttpResponse("Placeholder for Verb Browser")
+    """
+    View function for home page
+    """
+    num_verbs = Verb.objects.all().count()
+    num_examples = Example.objects.all().count()
+    return render(
+        request,
+        'ruskeyverbs/index.html',
+        context={'num_verbs': num_verbs, 'num_examples': num_examples}
+    )
+
+
+@login_required
+def VerbListPerUser(request):
```

Comment:
This seems weird and I'm having a hard time entirely understanding the point here. Is the purpose of the three different lists just so you can display them in three columns? If so there's definitely better ways to achieve that.

I also think that you can probably be doing more work in the db here than you are. Seems like you are doing a group by operation which is another thing SQL databases are really good at. Take a look at the annotate method in Django.